src/paxos/learner.ml

```diff
@@ -0,0 +1,203 @@
+(*
+Copyright (2010-2014) INCUBAID BVBA
```

Comment:
we should probably move to a copyright notice like the one in Alba. I think it's not acceptable to attribute copyright to someone who didn't create it.

Reply:
Ok.